snuba/cli/consumer.py

```diff
@@ -133,56 +134,77 @@ def consumer(
     profile_path: Optional[str] = None,
     cooperative_rebalancing: bool = False,
 ) -> None:
+    try:
```

Comment:
could the try be around `consumer.run()` only?

Reply:
I had it this way originally, but creating the `consumer_builder` object itself could crash the startup of the consumer. We basically want the entire logic of running the `snuba consumer` CLI captured in case anything goes wrong.